ui/src/org.ts

```diff
@@ -0,0 +1,342 @@
+import * as svelteStore from "svelte/store";
```

Comment:
On the organization of code in this module and `theGraphApi` I’d propose the following: Let’s split it into a stateless data fetching layer and the stateful “service” layer.

The data fetching layer would not contain any stateful code. It would not make use of stores, notifcations, module-level variables, etc. It’s primary purpose would be to export functions that get or update org-related data from the sources. (Maybe we separate data fetchers for the graph, contracts, and gnosis safe.)

On top of this layer would be the layer that keeps state and interacts with the UI (e.g. through notifications.)

Reply:
https://github.com/radicle-dev/radicle-upstream/issues/2025